feather/common/src/events.rs

```diff
@@ -58,3 +59,12 @@ pub struct ChunkLoadEvent {
 pub struct ChunkLoadFailEvent {
     pub position: ChunkPosition,
 }
+
+#[derive(Debug, Clone)]
+pub struct TablistHeaderFooter {
+    pub header: Text,
+    pub footer: Text,
+}
```

Comment:
`feather_common::events` is for events, while `TablistHeaderFooter` is a resource. I'm not sure what file would fit best for this small struct, but definitely not `events.rs`

Reply:
Where does it need to be for plugins to get to it?

Reply:
`quill/common`. Quill doesn't support resources though.

Reply:
I'm asking this because I want to know if `TablistHeaderFooter` can be in tablist.rs.

Reply:
You can create `feather/common/src/tablist.rs` or `quill/common/src/tablist.rs`, but not in `feather/server`. No other crate can access `feather-server`.